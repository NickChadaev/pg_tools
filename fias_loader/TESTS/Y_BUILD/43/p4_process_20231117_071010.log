2023-11-17 07:10:10 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_43 postgres /home/n.chadaev@abrr.local/Y_BUILD/43/stage_gar_c_43.csv 2023-11-13
--------------------------------------------------------
[07:10:10] ... Кировская обл., id_region: 42, Execute: /home/n.chadaev@abrr.local/Y_BUILD/43/stage_gar_c_43.csv, process_id:25345
[07:10:10] ... Start process
[07:10:10] ...  -- Version
[07:10:10] ...  -- Очистка от данных
[07:10:10] ...  -- Убираем индексы
[07:10:10] ...  -- Set UNLOGGED
[07:10:10] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_APARTMENT_TYPES_20231113_78e8cff6-619c-4b2e-806f-6c3c18c41d8c.XML
[07:10:10] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDR_OBJ_TYPES_20231113_15f2ec37-c925-4252-b49a-5473d975a16f.XML
[07:10:11] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ROOM_TYPES_20231113_32952955-cd74-40c1-bc84-06d94fe9559d.XML
[07:10:11] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OPERATION_TYPES_20231113_5151274c-91f4-4286-aaff-fc37112b660a.XML
[07:10:11] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_PARAM_TYPES_20231113_fe767969-20f3-4044-9ef6-67b4cb2881ce.XML
[07:10:11] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_HOUSE_TYPES_20231113_507d1000-4460-4d2f-a5b1-2ab5b9cd8a26.XML
[07:10:11] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDHOUSE_TYPES_20231113_d99b31fe-7b05-400a-ac92-23919e58da07.XML
[07:10:11] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OBJECT_LEVELS_20231113_b5fa7b3d-ea26-40c9-8b3a-9b986261ea66.XML
[07:10:11] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_TYPES_20231113_b8068ed2-dd18-4365-abb7-3dbd98698c35.XML
[07:10:11] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_KINDS_20231113_2f0f38c6-d598-4fd7-b325-2aa6aca29ed2.XML
[07:10:11] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/43/AS_REESTR_OBJECTS_20231113_d6fba411-4893-49ac-911b-b554ce3c5d01.XML
[07:23:45] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/43/AS_ADDR_OBJ_20231113_08a27cb4-f739-4c5e-9a90-e9ad3ceee25a.XML
[07:24:08] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/43/AS_HOUSES_20231113_ce00d8b6-afc0-4f17-8d66-14d95365e972.XML
[07:32:39] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/43/AS_STEADS_20231113_a0bdb698-7217-4ea6-90a0-919591c93998.XML
[07:35:12] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/43/AS_APARTMENTS_20231113_000812ba-fcfb-4272-a416-e6aebc95e0f4.XML
[07:42:23] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/43/AS_ROOMS_20231113_b9dd4f39-66ec-41e3-89ba-8652db7644a8.XML
[07:42:28] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/43/AS_CARPLACES_20231113_0e846eca-f408-4d88-b17f-588f11c800b2.XML
[07:42:29] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/43/AS_ADM_HIERARCHY_20231113_48e759bf-ba79-43d0-ae46-4e05079d332c.XML
[07:57:19] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/43/AS_MUN_HIERARCHY_20231113_c5681b3d-5f29-45e2-b1a1-7d35f85837e3.XML
[08:14:08] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/43/AS_CHANGE_HISTORY_20231113_938b2be5-c65e-4baf-b8e3-71241d07305f.XML
[08:58:00] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/43/AS_ADDR_OBJ_DIVISION_20231113_1d71d550-2102-4b54-a861-57b97b5dd654.XML
[08:58:00] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/43/AS_ADDR_OBJ_PARAMS_20231113_1bfa50fc-52b6-4711-8335-30456497e3ba.XML
[09:02:40] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/43/AS_STEADS_PARAMS_20231113_1b913dae-991d-4769-8f82-3d03dfb32c81.XML
[09:40:12] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/43/AS_HOUSES_PARAMS_20231113_8a974db8-d190-4db6-a06d-138075747348.XML
[10:43:15] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/43/AS_APARTMENTS_PARAMS_20231113_8744f905-88fe-4355-aad4-ae5045a778bc.XML
[10:55:49] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/43/AS_ROOMS_PARAMS_20231113_720e40ba-92a6-457b-92bc-073f6ad03927.XML
[10:55:54] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/43/AS_CARPLACES_PARAMS_20231113_69781caf-fbce-4248-8fb8-6d2731e06b26.XML
[10:55:55] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/43/AS_NORMATIVE_DOCS_20231113_ffada182-3b02-4cea-89e0-15e8612970d1.XML
[10:58:18] ...  -- Data finish
[10:58:18] ...  -- Set LOGGED
[10:58:50] ... Stop process
--------------------------------------------------------
