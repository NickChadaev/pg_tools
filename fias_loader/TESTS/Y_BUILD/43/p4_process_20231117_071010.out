 save_gar_version 
------------------
               81
(1 row)

CALL
CALL
CALL
 set_gar_dt_create 
-------------------
                81
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 1135569
gar_fias.as_addr_obj: 27334
gar_fias.as_houses: 634314
gar_fias.as_steads: 209797
gar_fias.as_apartments: 576174
gar_fias.as_rooms: 5620
gar_fias.as_carplaces: 1265
gar_fias.as_adm_hierarchy: 1180640
gar_fias.as_mun_hierarchy: 1310742
gar_fias.as_change_history: 3644653
gar_fias.as_addr_obj_division: 75
gar_fias.as_addr_obj_params: 359363
gar_fias.as_steads_params: 2785485
gar_fias.as_houses_params: 4875606
gar_fias.as_apartments_params: 1027801
gar_fias.as_rooms_params: 6835
gar_fias.as_carplaces_params: 409
gar_fias.as_normative_docs: 188738
